courses/free-courses-en.md

```diff
@@ -1045,3 +1046,10 @@
 * [Web Development for Beginners - A Curriculum](https://github.com/microsoft/Web-Dev-For-Beginners) - Microsoft
 * [Web Development Tutorials for Beginners](https://www.youtube.com/playlist?list=PLoYCgNOIyGAB_8_iq1cL8MVeun7cB6eNc) - LearnCode.academy
 * [Web Information Retrieval](https://www.youtube.com/playlist?list=PLAQopGWlIcya-9yzQ8c8UtPOuCv0mFZkr) - L. Becchetti, A. Vitaletti (University of Sapienza Rome)
+
+
+### Web3
+
+* [Become a Web3 Developer](https://learnweb3.io) - Course for everyone (Basics - Advanced - Expert) level
+* [Learning Resources](https://web3.career/learn-web3) - Learn Web3 With These Free Resources
```

Comment:
Like in the past.... Has several links without relation between, so... move to Metalist section